Dockerfile

```diff
@@ -42,3 +42,8 @@ ENV DB_NAME=application-db \
 COPY django-entrypoint.sh /usr/local/bin/
 RUN chmod +x /usr/local/bin/django-entrypoint.sh
 ENTRYPOINT ["django-entrypoint.sh"]
+
+# install geckodriver needed for selenium tests
```

Comment:
Looks OK but I have 0 experience with this, just checked https://askubuntu.com/questions/870530/how-to-install-geckodriver-in-ubuntu

Reply:
During local build geckodriver is correctly installed